packages/ast-spec/package.json

```
@@ -32,13 +32,13 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc -b tsconfig.build.json && api-extractor run --local",
-    "clean": "tsc -b tsconfig.build.json --clean",
-    "postclean": "rimraf dist && rimraf coverage",
+    "clean": "tsc -b tsconfig.build.json --clean & yarn run postclean",
+    "postclean": "rimraf dist/ coverage/",
     "clean-fixtures": "rimraf -g \"./src/**/fixtures/**/snapshots\"",
-    "format": "prettier --write \"./**/*.{ts,mts,cts,tsx,js,mjs,cjs,jsx,json,md,css}\" --ignore-path ../../.prettierignore",
-    "lint": "npx nx lint",
+    "format": "yarn run -T format",
+    "lint": "yarn run -T nx lint",
     "test": "jest",
-    "check-types": "npx nx typecheck"
+    "check-types": "yarn run -T nx typecheck"
```

```suggestion
    "check-types": "yarn run --top-level nx typecheck"
```
For some reason `knip` does not like `yarn run -T` but it does accept `yarn run --top-level` which is a little strange considering they're `-T` is just an alias for `--top-level`.